pkg/apis/clusteroperator/v1alpha1/types.go

```diff
@@ -151,6 +159,10 @@ type ClusterStatus struct {
 	// For machine set hardware, see the status of each machine set resource.
 	Provisioned bool `json:"provisioned"`
 
+	// ProvisioningJob is the reference to the Job performing infrastructure provisioning
+	// for this cluster. It will be set while the infrastructure is being provisioned.
+	ProvisioningJob *corev1.LocalObjectReference `json:"provisioningJob,omitempty"`
```

Comment:
Is this the only job type?  Is this guaranteed to be nil when Provisioned is set to true?

Reply:
> Is this the only job type?
yes (for what we need right now)




> Is this guaranteed to be nil when Provisioned is set to true?

yes

Reply:
Can you add that to the godoc so it's clear?

Reply:
done